src/transformers/models/vision_text_dual_encoder/configuration_vision_text_dual_encoder.py

```diff
@@ -49,15 +49,15 @@ class VisionTextDualEncoderConfig(PretrainedConfig):
     Examples:
 
     ```python
-    >>> from transformers import ViTConfig, BertConfig, VisionTextDualEncoderConfig, VisionTextDualEncoderModel
+    >>> from transformers import VisionTextDualEncoderModel, VisionTextDualEncoderConfig, BertConfig, ViTConfig
```

Comment:
The idea is to keep the config before model importing. The original order is already good in this case. No need to change here.

Reply:
Okay, I will revert this change.